packages/ssx-sdk/src/core.ts

```diff
@@ -163,12 +164,23 @@ export class SSXConnected implements ISSXConnected {
   public async ssxServerNonce(params: Record<string, any>): Promise<string> {
     if (this.api) {
       let nonce;
+
       try {
+        const route =
+          this.config.providers?.server?.routes?.nonce ?? '/ssx-nonce';
+        const requestConfig =
```

Comment:
replace the if with the guard check implemented at ssx-core

Reply:
done!